backend/src/main/java/ai/verta/modeldb/ModelDBConstants.java

```diff
@@ -198,7 +198,7 @@
   // Telemetry Constants
   String TELEMETRY = "telemetry";
   String OPT_IN = "opt_in";
-  String TELEMENTRY_FREQUENCY = "frequency"; // frequency to share data in hours
+  String FREQUENCY = "frequency"; // frequency to share data in hours
```

Comment:
Let's keep this variable and use another one. They have very different meanings in the two contexts right now, as the comment itself says.

Reply:
constant reverted and created a new one for frequency